buildbot-host/buildmaster/master.cfg

```diff
@@ -504,17 +505,34 @@ for factory_name, factory in factories.items():
 # We need to create schedulers after the builders, because otherwise the build
 # name lists are not available yet.
 c['schedulers'] = []
+
+# Only build if any of the files in the Change match this regular expression
+openvpn_file_patterns = ["include/.*",
+                         "m4/.*",
+                         "src/compat/.*",
+                         "src/openvpn/.*",
+                         "tests/.*",
+                         "compat.m4",
+                         "config.h\..*",
+                         "configure.ac",
+                         "Makefile.am",
+                         "version.m4"]
```

Comment:
Need to include `doc/.*\.rst` and `doc/Makefile.am` for the man-page/HTML builds.
Otherwise I'm fine with the list now.

Reply:
I added `doc/Makefile.am` and `.*\.rst`. While most .rst files are under *doc*, I wanted to cover *Changes.rst* and *CONTRIBUTING.rst* as well.

Looks good now?

Reply:
I'll resolve the merge conflict. Missed it the last time.

Reply:
Rebased against latest master and tested. Seems to work as intended. A DependentScheduler does not get triggered if the change_filter of their corresponding SingleBranchScheduler ignored as change. Which is what I'd expect.